pages/articles/[filename].tsx

```diff
@@ -84,18 +89,56 @@ export default function ArticlesPage(
                 components={componentRenderer}
               />
             </div>
-            {data.articles.showSidebarPanel && (
+            {data.articles.sidebarPanel?.showSidebarPanel && (
               <div className="w-full px-16 lg:shrink lg:pl-16 lg:pr-0">
                 <SidebarPanel
-                  title={data.articles.sidebarPanel.title}
-                  description={data.articles.sidebarPanel.description}
-                  actionUrl={data.articles.sidebarPanel.actionUrl}
-                  actionText={data.articles.sidebarPanel.actionText}
+                  title={data.articles.sidebarPanel?.title}
+                  tinaFields={{
+                    title: tinaField(data.articles.sidebarPanel, "title"),
+                    description: tinaField(
+                      data.articles.sidebarPanel,
+                      "description"
+                    ),
+                  }}
+                  description={data.articles.sidebarPanel?.description}
+                  actionUrl={data.articles.sidebarPanel?.actionUrl}
+                  actionText={data.articles.sidebarPanel?.actionText}
                 />
               </div>
             )}
           </section>
         )}
+
+        {data.articles.callToAction?.showCallToAction && (
+          <CallToAction
+            animated={data.articles?.callToAction?.animated}
+            tinaFields={{
+              subTitle: tinaField(data.articles?.callToAction, "subTitle"),
+              buttonSubtitle: tinaField(
+                data.articles?.callToAction,
+                "buttonSubtitle"
+              ),
+            }}
+            subTitle={data.articles?.callToAction?.subTitle}
+            buttonText={data.articles?.callToAction?.buttonText}
+            buttonSubtitle={data.articles?.callToAction?.buttonSubtitle}
+          >
+            {data.articles?.callToAction?.title && (
+              <h1
+                className="mx-auto w-fit"
+                data-tina-field={tinaField(
+                  data.articles?.callToAction,
+                  "title"
+                )}
+              >
+                {data.articles?.callToAction?.title}
+              </h1>
```

Comment:
There is already a H1 on this page - I dont think this should be a heading at all because it doesn't have anything to do with the content structure

Reply:
Done - I've changed it to a level  2 style tag but kept the styling the same